x-pack/legacy/plugins/reporting/export_types/common/lib/screenshots/wait_for_render.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { HeadlessChromiumDriver as HeadlessBrowser } from '../../../../server/browsers/chromium/driver';
+import { LevelLogger as Logger } from '../../../../server/lib/level_logger';
+import { LayoutInstance } from '../../layouts/layout';
+
+export const waitForRenderComplete = async (
+  captureConfig: any,
+  browser: HeadlessBrowser,
+  layout: LayoutInstance,
+  logger: Logger
+) => {
+  logger.debug('waiting for rendering to complete');
+
+  return await browser
+    .evaluate({
+      fn: (selector, visLoadDelay) => {
+        // wait for visualizations to finish loading
```

Comment:
Could also move this to .map or another. Given that there's a few of these I think I could be convinced to do so in another PR...